packages/amplify-e2e-core/src/categories/storage.ts

```diff
@@ -537,6 +537,43 @@ export function updateS3AddTrigger(cwd: string, settings: any): Promise<void> {
   });
 }
 
+export function addS3StorageWithIdpAuth(projectDir: string): Promise<void> {
```

Comment:
Did you add this test for S3?

Reply:
Yes I added it for the e2e tests

Reply:
These have changes - did you ensure they pass?